### PR TITLE
Show Link list #11

### PR DESCRIPTION
**Pull request by djsowa**
LinkList component

### CODE REVIEW

File: apps/links/webpack.config.js [hunk 1]
```diff
@@ -52,6 +52,7 @@ module.exports = {
   ],
   devServer: {
     historyApiFallback: true,
-    contentBase: './',
+	disableHostCheck: true,
```
Comment (mg6maciej) [inline apps/links/webpack.config.js]: Tabs!

File: apps/links/webpack.config.js [hunk 2]
```diff
@@ -52,6 +52,7 @@ module.exports = {
...
+    contentBase: './'
```
Comment (mg6maciej) [inline apps/links/webpack.config.js]: Missing trailing comma.

Comment (djsowa) [inline apps/links/webpack.config.js]: Fixed

File: apps/links/webpack.config.js (hunk 1 above)
Comment (djsowa) [inline apps/links/webpack.config.js]: fixed

**Pull request merged**